app/views/deployments.html

```diff
@@ -28,8 +28,8 @@ <h1>
           <div ng-if="(showEmptyState || filterWithZeroResults) && deploymentConfigsLoaded">
             <div class="empty-state-message text-center">
               <div ng-if="showEmptyState">
-                <h2>No deployments.</h2>
-                <p>No deployments have been added to project {{projectName}}.</p>
+                <h2 translate>No deployments.</h2>
```

Comment:
Question re: consistency / moving forward to `components` over `directives`, would we want it to always be an element `<translate>` or are we ok with an attrib `<p translate>`?  Do we then switch these to `<p><translate>`?

Reply:
This comment relates to a number of elements in the PR.

Reply:
Well it's from angular-gettext, so it will be directive. That's how they implemented it.

I'd lean towards using it as an attribute to avoid a bunch of extra elements in our markup, but it can be used as an element.

https://angular-gettext.rocketeer.be/dev-guide/annotate/

I'm a little concerned that angular-gettext doesn't support Angular 2. If we update Angular, we'd need to find a different solution and rework all of the translations (in addition to everything else).

Reply:
I tend to prefer it as a filter, like [angular-translate](https://angular-translate.github.io/docs/#/guide) handles it.  It seems more like the kind of thing a filter would do, and that avoids the issue with nodes.

That said, there is a lot of praise for `angular-gettext` being easier to work with.

Reply:
@spadgett Angular-gettext dose not support Angular2. I haven't found a way to compatible Angular1 and Angular2 for the time being.If you upgrade to Angular2,we need to find other ways of translating.